build.gradle

```diff
@@ -3,6 +3,7 @@ buildscript {
 
     repositories {
         mavenCentral()
+        jcenter()
```

Comment:
What exact dependency resolution errors do you have? We have CI in place and also locally build is ok

Reply:
This is weird. In the morning the dependency resolve was not working on the following dependency:
```
https://plugins.gradle.org/m2/com/jfrog/bintray/com.jfrog.bintray.gradle.plugin/1.8.4/com.jfrog.bintray.gradle.plugin-1.8.4.pom
```

This was redirected to a different server, I think jfrog and was trying to download the JAR file. Right now the dependency resolve after the Gradle cache cleanup works correctly.

I will remove jcenter repo from this PR.

I already had similar issue in my previous work and the problem was with slow connections. It might be useful to have proxy cached versions of all remote repos.

Reply:
We have cached versions of all remote repos for bigger projects :)
teamcity-rest-client is seldom updated, so we prefer official servers here.